examples/Reranking_search_results_with_cross-encoders.ipynb

```diff
@@ -0,0 +1,805 @@
+       "      <td>Yes</td>\n",
+       "      <td>-0.474471</td>\n",
+       "      <td>0.622214</td>\n",
+       "      <td>0.622214</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>3</td>\n",
+       "      <td>how do bi-encoders work for sentence embeddings</td>\n",
+       "      <td>Evaluating the Construct Validity of Text Embe...</td>\n",
+       "      <td>No</td>\n",
+       "      <td>-0.000509</td>\n",
+       "      <td>0.999491</td>\n",
+       "      <td>0.000509</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>4</td>\n",
+       "      <td>how do bi-encoders work for sentence embeddings</td>\n",
+       "      <td>Learning Probabilistic Sentence Representation...</td>\n",
+       "      <td>Yes</td>\n",
+       "      <td>-0.252695</td>\n",
+       "      <td>0.776705</td>\n",
+       "      <td>0.776705</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   index                                            query  \\\n",
+       "0      0  how do bi-encoders work for sentence embeddings   \n",
+       "1      1  how do bi-encoders work for sentence embeddings   \n",
+       "2      2  how do bi-encoders work for sentence embeddings   \n",
+       "3      3  how do bi-encoders work for sentence embeddings   \n",
+       "4      4  how do bi-encoders work for sentence embeddings   \n",
+       "\n",
+       "                                            document prediction  logprobs  \\\n",
+       "0  SBERT studies Meaning Representations: Decompo...        Yes -0.000248   \n",
+       "1  Are Classes Clusters?: Sentence embedding mode...         No -0.000363   \n",
+       "2  Semantic Composition in Visually Grounded Lang...        Yes -0.474471   \n",
+       "3  Evaluating the Construct Validity of Text Embe...         No -0.000509   \n",
+       "4  Learning Probabilistic Sentence Representation...        Yes -0.252695   \n",
+       "\n",
+       "   probability  cleaned_probability  \n",
+       "0     0.999752             0.999752  \n",
+       "1     0.999637             0.000363  \n",
+       "2     0.622214             0.622214  \n",
+       "3     0.999491             0.000509  \n",
+       "4     0.776705             0.776705  "
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "output_df = pd.DataFrame(\n",
+    "    output_list, columns=[\"query\", \"document\", \"prediction\", \"logprobs\"]\n",
+    ").reset_index()\n",
+    "# Use exp() to convert logprobs into probability\n",
+    "output_df[\"probability\"] = output_df[\"logprobs\"].apply(exp)\n",
+    "# Reorder based on likelihood of being Yes\n",
+    "output_df[\"cleaned_probability\"] = output_df.apply(\n",
```

Comment:
`cleaned_probability` confused me at first - maybe call it `yes_probability` instead? I think that would be clearer to someone scanning the table and wondering why `probability` is dirty.

Or, an alternative way of doing this could be to:
- Return the top 2 logprobs instead of top 1
- Filter to yes rows

Then you don't need the complexity of a lambda function and two differently named probability columns

Reply:
Changed to `yes_probability` - tried top 2 logprobs but often got confounding ones even with `logit_bias` turned up for Yes and No i.e. I would get `No` and ` No` and the top two values.

Reply:
Oh, interesting. Maybe the boost wasn't high enough for ` Yes` to outweigh `No`?